scripts/post-processing/2-dim_reduction_analysis.R

```diff
@@ -41,12 +42,13 @@ option_list <- list(
               metavar = "character"),
   make_option(opt_str = c("-m", "--metadata"), type = "character",
               default = "none", help = "Path to metadata file that contains 
-              only the metadata variables you wish to test and label by.",
+              only the metadata variables you wish to test and label by. First
+              column must contain the sample names that are in the datasets",
               metavar = "character"),
   make_option(opt_str = c("-r", "--reduce"), type = "character",
-              default = "none", help = "Dimension reduction technique to use.
-              Options are: 'pca', 'tsne', or 'umap' If none is given, full 
-              datasets will be used.", metavar = "character"),
+              default = "pca", help = "Dimension reduction technique to use.
```

Comment:
You've made a change to the default behavior without a corresponding change to the documentation above.

Reply:
Just to make sure I know what you are referring to, you're talking about the default technique being PCA?

Reply:
Yes